docs/source/transforms.rst

```diff
@@ -328,9 +352,9 @@ Dictionary-based Composables
     :members:
     :special-members: __call__
 
-`RandUniformPatchd`
+`RandCenterSpatialCropd`
 ~~~~~~~~~~~~~~~~~~~~~
```

Comment:
this gives a warning 'WARNING: Title underline too short.'

Reply:
fixed in the latest patch.
Thanks.